manim/mobject/tabular.py

```diff
@@ -0,0 +1,1121 @@
+
+            class GetRowLabelsExample(Scene):
+                def construct(self):
+                    table = Tabular(
+                        [["First", "Second"],
+                        ["Third","Fourth"]],
+                        row_labels=[Text("R1"), Text("R2")],
+                        col_labels=[Text("C1"), Text("C2")])
+                    lab = table.get_row_labels()
+                    for item in lab:
+                        item.set_color(random_bright_color())
+                    self.add(table)
+        """
+
+        return VGroup(*self.row_labels)
+
+    def get_col_labels(self) -> VGroup:
+        """Return the column labels of the table.
+
+        Returns
+        --------
+        :class:`~.VGroup`
+            VGroup containing the column labels of the table.
+
+        Examples
+        --------
+
+        .. manim:: GetColLabelsExample
+            :save_last_frame:
+
+            class GetColLabelsExample(Scene):
+                def construct(self):
+                    table = Tabular(
+                        [["First", "Second"],
+                        ["Third","Fourth"]],
+                        row_labels=[Text("R1"), Text("R2")],
+                        col_labels=[Text("C1"), Text("C2")])
+                    lab = table.get_col_labels()
+                    for item in lab:
+                        item.set_color(random_bright_color())
+                    self.add(table)
+        """
+
+        return VGroup(*self.col_labels)
+
+    def get_labels(self) -> VGroup:
+        """Returns the labels of the table.
+
+        Returns
+        --------
+        :class:`~.VGroup`
+            :class:`~.VGroup` containing all the labels of the table.
+
+        Examples
+        --------
+
+        .. manim:: GetLabelsExample
+            :save_last_frame:
+
+            class GetLabelsExample(Scene):
+                def construct(self):
+                    table = Tabular(
+                        [["First", "Second"],
+                        ["Third","Fourth"]],
+                        row_labels=[Text("R1"), Text("R2")],
+                        col_labels=[Text("C1"), Text("C2")])
+                    lab = table.get_labels()
+                    colors = [BLUE, GREEN, YELLOW, RED]
+                    for k in range(len(colors)):
+                        lab[k].set_color(colors[k])
+                    self.add(table)
+        """
+        label_group = VGroup()
+        if self.top_left_entry is not None:
+            label_group.add(self.top_left_entry)
+        for label in (self.col_labels, self.row_labels):
+            if label is not None:
+                label_group.add(*label)
+        return label_group
+
+    def add_background_to_entries(self) -> "Tabular":
+        """Adds a black :class:`~.BackgroundRectangle` to each entry of the table."""
+        for mob in self.get_entries():
+            mob.add_background_rectangle(color=self.entries_background_color)
+        return self
```

Comment:
I was thinking more along the lines of passing in a paramter, sorry for not being more explicit 🙇

The code in the `__init__` would need to be changed too 😅  but github doesn't make it easy to do suggestions like that.

```suggestion
    def add_background_to_entries(self, color: Color = BLACK) -> "Tabular":
        """Adds a :class:`~.BackgroundRectangle` to each entry of the table."""
        for mob in self.get_entries():
            mob.add_background_rectangle(color=color)
        return self
```

Reply:
Yeah I did that in a stupid way, should be fixed now!